chrome/content/zotero-better-bibtex/db.coffee

```diff
@@ -80,17 +80,25 @@ Zotero.BetterBibTeX.DB = new class
     if !cacheReset && !keepCache
       cacheReset = @metadata.BetterBibTeX != Zotero.BetterBibTeX.release
       if cacheReset && confirmCacheResetSize && (@cache.data.length > confirmCacheResetSize || @serialized.data.length > confirmCacheResetSize)
-        cacheReset = Services.prompt.confirm(null, 'Drop Better BibTeX cache?', [
-          'You have upgraded BetterBibTeX. This usually means output generation for Bib(La)TeX has changed, and it is recommended to clear the cache in order for these changes to take effect.'
+        flagsForCacheResetDialog = prompts.BUTTON_POS_0 * prompts.BUTTON_TITLE_IS_STRING +
+                                   prompts.BUTTON_POS_1 * prompts.BUTTON_TITLE_IS_STRING
+        cacheReset = Services.prompt.confirm(
```

Comment:
I think confirmEx returns a bitmask, are you sure it returns a false-ish value for "Keep"?